torch/_inductor/fx_passes/serialized_patterns/_sfdp_pattern_21.py

```diff
@@ -42,47 +42,43 @@
 view_default_1 = CallFunction(aten.view.default, clone_default_1, Ignored(), _users=2)
```

Comment:
not really feedback, just curious: what are these files doing?

Reply:
These are like cached compiled patterns. Used to speedup compilation